## Multipliers for Delta tiers
## increases a measurements threshold to be added to the buffer by the given amount
delta_tiers: [1,2,3,4,5,6,7,8,9,10]

strapdown_topic: odometry/filtered/odom # To disable, set to None

### Buffer space configurations ###
buffer_space:
  capacity: 29 # Total capacity

### Process Noise Configurations ###
position_process_noise: 0.1
velocity_process_noise: 0.0

known_position_uncertainty: 1.0
unknown_position_uncertainty: 1000.0